WORKSPACE

```diff
@@ -117,8 +117,11 @@ haskell_nixpkgs_packageset(
     name = "hackage-packages",
     base_attribute_path = "haskellPackages",
     nix_file = "//tests:ghc.nix",
+    nixopts = [
```

Comment:
What for?

Reply:
Same as https://github.com/tweag/rules_haskell/pull/533#discussion_r243332618